src/main/java/frc/team2412/robot/Main.java

```diff
@@ -24,6 +22,6 @@ private Main() {
      * If you change your main robot class, change the parameter type.
      */
     public static void main(String... args) {
-        RobotBase.startRobot((Supplier<RobotBase>) Robot::getInstance);
+        RobotBase.startRobot(() -> Robot.instance);
```

Comment:
Really should switch this back to `RobotBase.startRobot(Robot::new);` from the WPILib examples and get rid of the Robot singleton entirely.